tracee/tracee_test.go

```diff
@@ -102,81 +112,16 @@ func TestReadArgFromBuff(t *testing.T) {
 		{
 			name: "sockAddrT",
 			input: []byte{12, //sockAddrT
+				0,    //TODO proper tag?
 				2, 0, //sa_family=AF_INET
 				0xFF, 0xFF, //sin_port=65535
 				0xFF, 0xFF, 0xFF, 0xFF, //sin_addr=255.255.255.255
 			},
-			expectedArg: "{'sa_family': 'AF_INET','sin_port': '65535','sin_addr': '255.255.255.255'}",
-		},
-		{
```

Comment:
- usage of argTag and argType related to 915a1cc?
- probably refer to [tracee/argprinters.go](https://github.com/ronaudinho/tracee/blob/1133d380643bb5af7f8850cbe9035cfb965b6f92/tracee/argprinters.go) for args